cmph/_utils.py

```
@@ -1,6 +1,22 @@
 import six
 from os.path import abspath, exists
 
+unicode = str if six.PY3 else unicode
+
 
 def is_file(obj):
     return isinstance(obj, six.string_types) and exists(abspath(obj))
+
+
+def convert_to_bytes(key):
+    is_unicrud = (six.PY3 and isinstance(key, six.string_types)) or \
+                 (six.PY2 and isinstance(key, unicode))
+
+    if is_unicrud:
+        return bytes(key.encode('utf8'))
+    elif isinstance(key, six.string_types):
+        return bytes(key)
+    elif isinstance(key, six.integer_types):
+        return bytes([key])
+    else:
+        return bytes(key)
```

so unicode gets converted to UTF8, strings get turned into bytes directly, integers are assumed to be characters (?), and everything else is just converted to bytes directly?

what valid cases are there for the `else` clause? is this a best-effort approach or would it be better to fail loudly?